runtime/dancebox/src/lib.rs

```diff
@@ -386,15 +386,6 @@ parameter_types! {
     pub const ExistentialDeposit: Balance = EXISTENTIAL_DEPOSIT;
 }
 
-/// A reason for placing a hold on funds.
```

Comment:
TODO: check in Fork tests this does not create issues with existing holds!

Reply:
We need a migration for this, running the fork tests has made me realize that the enums are not mapped equally!

Reply:
Migration added already